docs/user/ppl/admin/datasources.rst

```diff
@@ -136,6 +136,17 @@ Master Key config for encrypting credential information
     # Print the master key
     print("Generated master key:", master_key)
 
+Datasource Allow Hosts Config
```

Comment:
additional layer of security?

Reply:
This is to give customer a config to disallow any kind of ssrf attacks.